src/main/java/com/squareup/kotlinpoet/Util.kt

```diff
@@ -85,10 +85,10 @@ internal fun characterLiteralWithoutSingleQuotes(c: Char) = when {
 }
 
 /** Returns the string literal representing `value`, including wrapping double quotes.  */
-internal fun stringLiteralWithQuotes(value: String): String {
+internal fun stringLiteralWithQuotes(value: String, indent: String = ""): String {
   if (value.contains("\n")) {
     val result = StringBuilder(value.length + 32)
-    result.append("\"\"\"\n|")
+    result.append("\"\"\"\n" + indent + "|")
```

Comment:
String concatenation shouldn't occur in calls to `append` on a `StringBuilder`. Use multiple calls to `append`